src/components/SuggestionsList/SuggestionsList.tsx

```diff
@@ -68,16 +70,16 @@ export default class SuggestionsList extends React.PureComponent<SuggestionsList
       <div className={join(['y-suggestions-list', 'y-hc-border', className, classNames.root])}>
         {hasResults && (
           <LayoutList direction="vertical">
-            {groupedItems.map((group: SuggestionItemGroupProps) => (
+            {groupedItems.map((group: SuggestionItemGroup) => (
```

Comment:
this is a fairly miserable mount of nesting - can this be pulled out into a this.getGroupedItem or whatever?

Reply:
i agree, i'll update these

Reply:
done